DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Validators/NullValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+using FluentValidation.Results;
+
+namespace GitHubExtension.Security.WebApi.Library.Validators
+{
+    sealed class NullValidator<T> : AbstractValidator<T>
```

Comment:
When I removed IServiceProvider I faced with problem that entity without own validators generate runtime error. So we need implement NullValidator for handling this situation
https://simpleinjector.readthedocs.org/en/latest/advanced.html#registration-of-open-generic-types